package.json

```diff
@@ -38,7 +38,7 @@
     "test:e2e:all": "yarn test:e2e:clean && yarn test:e2e; yarn test:e2e:report;",
     "test:e2e:kind": "CYPRESS_dexUrl=http://dex-twodotoh.a.influxcloud.dev.local CYPRESS_baseUrl=https://twodotoh.a.influxcloud.dev.local CYPRESS_password=password cypress open --browser chrome --config chromeWebSecurity=false",
     "test:e2e:remocal": "CYPRESS_dexUrl=https://dex-${NS}.remocal.influxdev.co CYPRESS_baseUrl=https://${NS}.remocal.influxdev.co CYPRESS_password=password cypress open --browser chrome --config chromeWebSecurity=false",
-    "test:circleci": "yarn run test:ci --maxWorkers=2",
+    "test:circleci": "yarn run test:ci --runInBand",
```

Comment:
Is there a reason we'd want these to run in serially instead?

Reply:
@abshierjoel it turns out that our jest tests were running out of memory. The general way to fix it on the internet is to set `maxWorkers` <= the number of CPUs your circle instance has and go from there. Since setting `maxWorkers` to 2 was causing problems, I dropped the number down to `1`, which is an alias for `runInBand`.

I talked with Gene about this. This isn't necessarily the best way to do it - there are tradeoffs. Lowering the number of CPUs will make the tests run for longer which will cost more money, but we'll still be on a relatively cheap CPU plan. The alternative is to increase the number of CPUs for this circleci job which will also cost more money. If this ends up taking too long or being too expensive, we can increase the resources available to our circle instance.